src/naemon/defaults.h

```diff
@@ -47,6 +47,7 @@
 #define DEFAULT_LOG_CURRENT_STATES				1	/* log current service and host states after rotating log */
 #define DEFAULT_LOG_EXTERNAL_COMMANDS				1	/* log external commands */
 #define DEFAULT_LOG_PASSIVE_CHECKS				1	/* log passive service checks */
+#define DEFAULT_log_global_notifications			1	/* log global notifications */
```

Comment:
Can we uppercase this please

Reply:
i am pretty sure it was uppercase initially :-) Will fix it again